pyproject.toml

```diff
@@ -55,11 +57,19 @@ m3gnet = [
 sevennet = [
     "sevenn == 0.10.3",
 ]
+fairchem = [ 
+    "fairchem-core == 1.4.0",
```

Comment:
If we want the version with stresses, run `uv add git+https://github.com/FAIR-Chem/fairchem#subdirectory=packages/fairchem-core --rev omat24` (this will need to be resolved by uv, so we need to deal with torch/e3nn first, but I can also figure out what it would look like it it helps)

Reply:
Now that the conflicts have been resolved, I think we'll keep the standard PyPI installation, but add a note in the docs for installing this branch, along with notes about the torch extras needed.